amp/WEB-INF/src/org/digijava/module/aim/dbentity/AmpFiscalCalendar.java

```diff
@@ -23,25 +24,27 @@
 
 public class AmpFiscalCalendar implements Serializable, Identifiable, OrgProfileValue, CalendarConverter {
     
-    //IATI-check: to be ignored
-//  @Interchangeable(fieldTitle="Fiscal Calendar ID")
+    @JsonProperty("id")
     private Long ampFiscalCalId;
-//  @Interchangeable(fieldTitle="Start Month Number")
+    
+    @JsonProperty("start_month_num")
```

Comment:
Please use kebab-case for json properties in API for new fields (and in general when adding new fields)